ui/src/app/applications/components/application-resources-diff/application-resources-diff.scss

```diff
@@ -6,9 +6,7 @@
         text-align: right;
         label {
             padding-right: 2em;
-            @include themify($themes) {            
-                color: themed('text-2');
-            }
+            color: $argo-color-gray-8;
```

Comment:
Question: does this apply for light mode only?

Reply:
Yeah, it is not related to transition, just a small fix related to compact diff menu
![image](https://user-images.githubusercontent.com/17771352/226893326-bea3877e-4ec8-47fa-a083-bc0acc575656.png)

Reply:
It does relate to dark/light mode change